metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-    "title": "CLDF dataset of the supplementary materials to "Subgrouping in a ‘dialect continuum’: A Bayesian phylogenetic analysis of the Mixtecan language family" (Auderset et al. 2023. JoLE)",
+    "title": "CLDF dataset of the supplementary materials to \"Subgrouping in a ‘dialect continuum’: A Bayesian phylogenetic analysis of the Mixtecan language family\" (Auderset et al. 2023. JoLE)",
     "url": "https://github.com/lexibank/mixtecansubgrouping",
     "license": "CC-By-4.0",
-    "citation": "Auderset, Sandra, Simon J. Greehnhill, Christian T. DiCanio, Eric W. Campbell. 2023. CLDF data set of Supplementary Materials to "Subgrouping in a `dialect continuum': A Bayesian phylogenetic analysis of the Mixtecan language family". Journal of Language Evolution, tbd. https://doi.org/10.5281/zenodo.6513506"
+    "citation": "Auderset, Sandra, Simon J. Greehnhill, Christian T. DiCanio, Eric W. Campbell. 2023. CLDF data set of Supplementary Materials to \"Subgrouping in a `dialect continuum': A Bayesian phylogenetic analysis of the Mixtecan language family\". Journal of Language Evolution, tbd. https://doi.org/10.5281/zenodo.6513506"
```

Comment:
No, that is not the CLDF citation, we want the citation of the paper here, plain, ideally with DOI (you can often derive it in JOLE from your number assigned to the paper upon submission). So please provide the citation of the forthcoming paper here.

Reply:
I checked the ID/number associated with my manuscript but it does not look like the DOI format of the published papers. I will add it as soon as I get it - is that okay? I adjusted the citation of the paper otherwise.